### PR TITLE
Logging Improvements for CloneWorkspace

### PR DESCRIPTION
**Pull request by jaycarlton**
Plus standardized the logging around a few test fixtures. We now indicate which errors are expected versus not.

### CODE REVIEW

File: integration/src/main/java/scripts/testscripts/CloneWorkspace.java [hunk 1]
```diff
@@ -124,6 +132,11 @@ protected void doSetup(
             getWorkspaceId(),
             sharedBucketSourceResourceName,
             CloningInstructionsEnum.RESOURCE);
+    //    logger.info(
```
Comment (yuhuyoyo) [inline integration/src/main/java/scripts/testscripts/CloneWorkspace.java:135]: did you mean to include this?

Comment (jaycarlton) [reply integration/src/main/java/scripts/testscripts/CloneWorkspace.java:135]: nope

**Review by yuhuyoyo — APPROVED** [review]

File: integration/src/main/java/scripts/testscripts/CloneWorkspace.java [hunk 2]
```diff
@@ -430,101 +550,178 @@ protected void doUserJourney(
...
-    logger.info(privateDatasetDetails.toString());
-    assertEquals(CloneResourceResult.FAILED, privateDatasetDetails.getResult());
-    assertEquals(CloningInstructionsEnum.RESOURCE, privateDatasetDetails.getCloningInstructions());
-    assertEquals(ResourceType.BIG_QUERY_DATASET, privateDatasetDetails.getResourceType());
-    assertEquals(StewardshipType.CONTROLLED, privateDatasetDetails.getStewardshipType());
-    assertNull(privateDatasetDetails.getDestinationResourceId());
-    assertNotNull(privateDatasetDetails.getErrorMessage());
+                .findFirst(),
+            "Private dataset clone results located.");
+    logger.info("Preivate dataset (expected failure): {}", privateDatasetDetails);
```
Comment (zloery) [inline integration/src/main/java/scripts/testscripts/CloneWorkspace.java:573]: nit: private

**Review by zloery — APPROVED** [review]
Thanks for this!

File: integration/src/main/java/scripts/testscripts/CloneWorkspace.java [hunk 3]
```diff
@@ -556,7 +764,11 @@ private static void assertDatasetHasNoTables(
     final TableResult listTablesResult = bigQueryClient.query(listTablesQuery);
     final long numRows =
         StreamSupport.stream(listTablesResult.getValues().spliterator(), false).count();
-    assertEquals(0, numRows);
+    assertEquals(0, numRows, "Expected zero tables for COPY_DEFINITION dataset");
+    logger.info(
```
[the remark this answers is not in the document]
Comment (jaycarlton) [reply integration/src/main/java/scripts/testscripts/CloneWorkspace.java:757]: That's deliberate. One's an assertion and the other is a log statement, so there's only duplication when it fails.

**Pull request merged**